tests/test_database_manager.py

```diff
@@ -0,0 +1,91 @@
+"""
+test_database_manager.py - UT for the databaseManager
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+This module is built on top of the Pydle system.
+"""
+import pytest
+import pyodbc
+from Modules.database_manager import DatabaseManager
+
+
+@pytest.fixture(scope="session")
+def prepare_dbm(dbm_fx):
+    """
+    Drops all the tables
+    """
+    #yield
+    table_names = ("testtablehas",
+                   "testtableselect",
+                   "testtablecreate",
+                   "testtableinsert",
+                   "testtableupdate",
+                   "testtabledelete",
+                   )
+    for name in table_names:
+        try:
+            dbm_fx.cursor.execute(f"DROP TABLE {name};")
+        except (pyodbc.ProgrammingError, pyodbc.OperationalError, pyodbc.DataError):
+            pass
+
+
+# noinspection PyProtectedMember
+@pytest.mark.asyncio
+@pytest.mark.database
+@pytest.mark.usefixtures("prepare_dbm")
+class TestStuff(object):
+    
+    async def test_has_table(self, dbm_fx):
+        if await dbm_fx.has_table("testtablehas"):
+            await dbm_fx.drop_table("testtablehas")
+        assert not await dbm_fx.has_table("testtablehas")
+        await dbm_fx.create_table("testtablehas", {"string1": "VARCHAR"})
+        assert await dbm_fx.has_table("testtablehas")
+
+    async def test_select_row(self, dbm_fx):
+        if not await dbm_fx.has_table("testtableselect"):
+            await dbm_fx.create_table("testtableselect", {"string1": "VARCHAR"})
+        await dbm_fx.insert_row("testtableselect", ("thest",))
+
+        tmp = await dbm_fx.select_rows("testtableselect", "AND", {"string1": "thest"})
+        print(tmp)

```

Comment:
This should be removed before merging :wink:

Reply:
Good eyes... Stupid debug stuff without a FIXME